packages/commonwealth/server/workers/commonwealthConsumer/commonwealthConsumer.ts

```diff
@@ -70,6 +71,7 @@ export async function setupCommonwealthConsumer(): Promise<void> {
   const contestWorkerSubRes = await brokerInstance.subscribe(
     BrokerSubscriptions.ContestWorkerPolicy,
     ContestWorker(),
+    buildRetryStrategy(undefined, 20_000),
```

Comment:
Just so I'm understanding correctly– this is using the default config (3 retries 2s delay), but overriding the delay to 20s?

Also, is it possible to use exponential backoff instead of linear? Not strictly required but it would be great for some things like RPC rate limiting.

Reply:
Rascal doesn't natively support exponential backoff but what you could do is generate an array of republish strategies which achieves the same thing i.e.:
```
[{ strategy: 'republish', defer: 20_000, attempts: 1}, { strategy: 'republish', defer: 40_000, attempts: 1}, {strategy: 'republish', defer: 120_000, attempts: 1}, ..., { strategy: 'nack' }];
```

Reply:
If this is something we want I recommend we merge the PR as is for now and then open a separate ticket for modifying the strategy to exponential backoff.

Reply:
And to answer your initial question, yes, `buildRetryStrategy(undefined, 20_000)` uses the default config and just increases the defer interval to 20 seconds.